src/blueprints/tv2/timeline-object-factories/tv2-atem-video-mixer-timeline-object-factory.ts

```diff
@@ -285,12 +290,17 @@ export class Tv2AtemVideoMixerTimelineObjectFactory implements Tv2VideoMixerTime
       return
     }
 
-    const atemMeTimelineObject: AtemMeTimelineObject = blueprintTimelineObject as AtemMeTimelineObject
-    return  atemMeTimelineObject.content.me.input
+    const atemMeTimelineObject: AtemMixEffectTimelineObject = blueprintTimelineObject as AtemMixEffectTimelineObject
+    if (atemMeTimelineObject.content.me.type === AtemMixEffectType.TRANSITION) {
+      return atemMeTimelineObject.content.me.input
+    } else {
+      return atemMeTimelineObject.content.me.programInput
+    }
```

Comment:
Just a pet peeve of mine 😄 You can do without the else, since you have a return in the if.

```suggestion
    }
    return atemMeTimelineObject.content.me.programInput
```

Reply:
It has been updated